packages/notifi-react-card/lib/context/NotifiContext.tsx

```diff
@@ -52,6 +52,14 @@ export type AcalaParams = Readonly<{
   signMessage: AcalaSignMessageFunction;
 }>;
 
+export type OptimismParams = Readonly<{
+  walletBlockchain: 'OPTIMISM';
+  accountAddress: string;
+  walletPublicKey: string;
+  signMessage: AcalaSignMessageFunction;
```

Comment:
why AcalaSignMessage instead of `Uint8SignMessageFunction`, which is used by other EVM stuff?

Reply:
sorry - fat finger. Resolving.